Seasonal/HarvestDay/MergeShops/EbilCorpHQMerge.cs

```diff
@@ -25,6 +26,7 @@ public void ScriptMain(IScriptInterface Bot)
         Core.BankingBlackList.AddRange(NeededItems);
         Core.SetOptions();
 
+        HarvestDay.EbilCorpHQ();
```

Comment:
Please move this inside the merge function, script main must remain empty as per format